services/actions-handler/go.mod

```diff
@@ -1,17 +1,26 @@
 module github.com/uselagoon/lagoon/services/actions-handler
 
-go 1.16
+go 1.18
 
 require (
-	github.com/cheekybits/is v0.0.0-20150225183255-68e9c0620927 // indirect
-	github.com/cheshir/go-mq v1.0.2
+	github.com/cheshir/go-mq v1.2.0
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
-	github.com/fsouza/go-dockerclient v1.7.3 // indirect
 	github.com/machinebox/graphql v0.2.3-0.20181106130121-3a9253180225
+	gopkg.in/matryer/try.v1 v1.0.0-20150601225556-312d2599e12e
+)
+
+require (
+	github.com/NeowayLabs/wabbit v0.0.0-20210927194032-73ad61d1620e // indirect
+	github.com/cheekybits/is v0.0.0-20150225183255-68e9c0620927 // indirect
+	github.com/fsouza/go-dockerclient v1.7.3 // indirect
+	github.com/google/uuid v1.3.0 // indirect
 	github.com/matryer/is v1.4.0 // indirect
 	github.com/matryer/try v0.0.0-20161228173917-9ac251b645a2 // indirect
+	github.com/pborman/uuid v1.2.1 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/rabbitmq/amqp091-go v1.4.0 // indirect
+	github.com/streadway/amqp v1.0.0 // indirect
 	github.com/tiago4orion/conjure v0.0.0-20150908101743-93cb30b9d218 // indirect
-	gopkg.in/matryer/try.v1 v1.0.0-20150601225556-312d2599e12e
 )
 
 // Fixes for AppID
```

Comment:
we'll have to check whether these fixes have applied, given the replace is for a different version

Reply:
they aren't :D we should maybe just stop using the appid stuff, worth investigating as part of `logs2notifications` more than solving here